public/assets/plugins/inline_style.min-d198d50f010b863e8d5aedcef1698ca118ec0d8a7927a2fa5ab870e54baacc50.js

```diff
@@ -0,0 +1,8 @@
+/*!
+ * froala_editor v2.0.5 (https://www.froala.com/wysiwyg-editor)
+ * License https://froala.com/wysiwyg-editor/terms
+ * Copyright 2014-2015 Froala Labs
+ */
+
+
+!function(a){"function"==typeof define&&define.amd?define(["jquery"],a):"object"==typeof module&&module.exports?module.exports=function(b,c){return void 0===c&&(c="undefined"!=typeof window?require("jquery"):require("jquery")(b)),a(c),c}:a(jQuery)}(function(a){"use strict";a.extend(a.FroalaEditor.DEFAULTS,{inlineStyles:{"Big Red":"font-size: 20px; color: red;","Small Blue":"font-size: 14px; color: blue;"}}),a.FroalaEditor.PLUGINS.inlineStyle=function(b){function c(c){""!==b.selection.text()?b.html.insert(a.FroalaEditor.START_MARKER+'<span style="'+c+'">'+b.selection.text()+"</span>"+a.FroalaEditor.END_MARKER):b.html.insert('<span style="'+c+'">'+a.FroalaEditor.INVISIBLE_SPACE+a.FroalaEditor.MARKERS+"</span>")}return{apply:c}},a.FroalaEditor.RegisterCommand("inlineStyle",{type:"dropdown",html:function(){var a='<ul class="fr-dropdown-list">',b=this.opts.inlineStyles;for(var c in b)a+='<li><span style="'+b[c]+'"><a class="fr-command" data-cmd="inlineStyle" data-param1="'+b[c]+'" title="'+this.language.translate(c)+'">'+this.language.translate(c)+"</a></span></li>";return a+="</ul>"},title:"Inline Style",callback:function(a,b){this.inlineStyle.apply(b)}}),a.FroalaEditor.DefineIcon("inlineStyle",{NAME:"paint-brush"})});
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected an identifier and instead saw '=='.<br>Expected an operator and instead saw 'typeof'.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'c' is already defined.<br>Expected an identifier and instead saw '!=='.<br>Mixed double and single quotes.<br>Expected '{' and instead saw 'a'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>'define' is not defined.<br>'require' is not defined.